taskvine/src/tools/vine_plot_txn_log

```diff
@@ -565,7 +587,11 @@ class TxnPlot:
     def set_legend(self, fig):
         if self.opts.legend != 'none':
             handles = [mpl.patches.Patch(color=c, label=l) for l,c in self.legend.items()]
-            fig.legend(loc=self.opts.legend, handles=handles)
+
+            fontsize = 'medium'
+            if args.fontsize:
+                fontsize = args.fontsize
+            fig.legend(loc=self.opts.legend, handles=handles, fontsize=fontsize)
```

Comment:
Use fontsize=args.fontsize.   You don't need the `if` nor the variable `fontsize`, as medium is already the default.